share/functions/realpath.fish

```diff
@@ -6,8 +6,10 @@
 # If we see that there is an external command by that name we just return. That will cause fish to
 # run the external command. On the other hand, if an external command isn't found we define a
 # function that will provide fallback behavior.
-if not type -q -P realpath
-    function realpath --description 'fallback realpath implementation'
-        builtin fish_realpath $argv[-1]
-    end
+if not command -v realpath > /dev/null
```

Comment:
FWIW, I think we should be using `command -s` rather than `command -v` in the scripts we ship since our command(1) man page documents `-s` as a first class flag and `-v` as being recognized only for compatibility with other implementations.